release-notes/2025-08/README.md

```diff
@@ -0,0 +1,179 @@
+# SystemLink Enterprise 2025-08 Release Notes
+
+The 2025-08 release for SystemLink Enterprise has been published to
+<https://downloads.artifacts.ni.com>. This update includes new features, bug
+fixes, and security updates. Work with your account representative to obtain
+credentials to access these artifacts. If you are not upgrading from the
+previous release, refer to past release notes to ensure you have addressed all
+required configuration changes.
+
+## New Features and Behavior changes
+
+- Support for Server-Side Encryption through the Key Management Service (AWS
+  SSE-KMS). For more information, refer to
+  [Configuring File Storage](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/configuring-file-storage.html).
+- `dataframeservice:1.20.72`
+  - The DataFrame Service now limits the amount of property metadata that tables
+    and columns can contain. This limit is configurable from the Helm values via
+    `dataframeservice.ingestion.maxPropertyMetadataLength`.
+  - This limit sets the maximum total character count for all property metadata
+    keys and values across the table and its columns.
+  - The default limit is 500000 characters. An
+    `HTTP 400 error: PropertyMetadataTooLarge` is thrown if the length of the
+    table's summed property metadata exceeds this value.
+  - This change mitigates database and query performance degradation.
+  - This change mitigates the likelihood of hitting the service database's 16 MB
+    document size limit.
+  - This limit is in addition to limits to the number of properties an
+    individual table or column may have.
+- `testmonitorservice:0.37.42`
+  - Introduced new Helm value named `testmonitor.queryRequestTimeoutInSeconds`,
```

Comment:
Suggestion:
'Introduced new `testmonitor.queryRequestTimeoutInSeconds` Helm value. Use this value to set the timeout limit for query requests.'